src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {
+    Red,

```

Comment:
fix common mistake
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values